packages/react-components/src/components/VideoGallery/DefaultLayout.tsx

```diff
@@ -78,9 +78,10 @@ export const DefaultLayout = (props: DefaultLayoutProps): JSX.Element => {
    * For some components which do not strictly follow the order of the array, we might
    * re-render the initial tiles -> dispose them -> create new tiles, we need to take care of
    * this case when those components are here
+   * Note: If for some reason the current number of videos rendered is more than the maximum allowed, 4 videos will be rendered
```

Comment:
Nit: should this be something along the lines of: 'If for some reason the current number of videos rendered is more than the maximum allow only the max number will show, if that max is less than 4 we will render 4 videos'?

Reply:
I don't think we should be using a minimum of 4 here. These video tile indices to render for the overflow gallery. If we are already rendering the maximum in the grid view we should not be rendering a minimum of 4.